### DISCUSSION TITLE
config_order for base element

### DISCUSSION THREAD

**Discussion by RisingSunLight42** [lowlighter/metrics#874] [Q&A] [answered]
To explain my problem, here what I would like to do : 

I would like to put in a specific order different plugin, and first `base.header`.
But, after that, I want to place the other attributes from base, so `base.activity`, etc.
When I do that, the element is ordered at the bottom, like when we do not specify the placement 😢

❓ Can we place specifically the other attributes like `base.activity` or `base.community` ?

**Comment by lowlighter** [comment, marked as answer]
Excluding the `introduction`, these are the handle for the base sections plugin ordering:
https://github.com/lowlighter/metrics/blob/803ffb0656239ded5f188ce9e324bb34c55044f5/source/templates/classic/partials/_.json#L2-L5

So as you can see, `base.activity` and `base.community` are actually combined into `base.activity+community` (because in the template it's actually the same file. 

For this same reason it's not possible to directly to set community before activity though with [`extras_css`](https://github.com/lowlighter/metrics/tree/master/source/plugins/core#-custom-css-styling) and a good selector you should be able to target it and use `flex-direction: row-reverse;` if you wish to reverse the order of it.

For anything more complicated, the best solution is probably to fork this repository, patch the templates as you wish and use your fork instead in your workflow 👍

**Reply by RisingSunLight42** [reply]
Wow ! Thanks you for your quick response !
Is tht written on the repository ? If not, can I suggest to write this specification somewhere ? Because it's not natural to write `base.activity+community` (well, for me at least). 😅

**Reply by lowlighter** [reply]
No it wasn't written, I added a note to the "ordering content" section 👍 
Thanks a lot for your feedback, this was indeed not very clear 😅
